### PR TITLE
chore(weather/current-city): Use testing-lib

### PR DESCRIPTION
**Pull request by jnv**
## Description

Use testing-lib consistently for weather/current-city. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: capabilities/weather/current-city/maps/current-city.ts [hunk 1]
```diff
@@ -1,40 +1,30 @@
...
-      client = new SuperfaceClient();
-      profile = await client.getProfile('weather/current-city');
-      provider = await client.getProvider(providerName);
-    });
+    let superface: SuperfaceTest;
 
-    it('should find provider', async () => {
-      expect(provider).not.toBeUndefined();
+    beforeEach(() => {
+      superface = new SuperfaceTest();
```
Comment (martinalbert) [inline capabilities/weather/current-city/maps/current-city.ts:13]: You can set up profile and provider here if there are no test runs with different profile or provider.

```suggestion
      superface = new SuperfaceTest({
        profile: 'weather/current-city',
        provider: providerName,
      });
```

File: capabilities/weather/current-city/maps/current-city.ts [hunk 2]
```diff
@@ -1,40 +1,30 @@
...
+        it('returns a weather data', async () => {
+          const result = await superface.run({
+            profile: 'weather/current-city',
+            provider: providerName,
+            useCase: 'GetCurrentWeatherInCity',
+            input: { city: 'Prague,CZ' },
+          });
+          const weatherData = result.unwrap();
+
+          expect(weatherData).toEqual({
```
Comment (martinalbert) [inline capabilities/weather/current-city/maps/current-city.ts:27]: Why not use snapshots?

**Review by martinalbert — APPROVED** [review]

File: capabilities/weather/current-city/maps/current-city.ts (hunk 2 above)
Comment (jnv) [reply capabilities/weather/current-city/maps/current-city.ts:27]: I guess I could. :wink:

**Comment by jnv** [comment]
Thanks for the review @martinalbert!

**Pull request merged**